waspc/src/Wasp/AppSpec/Entity.hs

```diff
@@ -38,3 +40,16 @@ getFields = fields
 
 getPslModelBody :: Entity -> PslModel.Body
 getPslModelBody = pslModelBody
+
+getPrimaryField :: Entity -> Maybe PslModel.Field
+getPrimaryField entity = find ((attrNameAssociatedWithPrimaryField `elem`) . getFieldAttrNames) pslFields
+  where
+    pslFields = [field | (PslModel.ElementField field) <- pslElements]
+    (PslModel.Body pslElements) = getPslModelBody entity
+
+    getFieldAttrNames :: PslModel.Field -> [String]
+    getFieldAttrNames PslModel.Field {_attrs = attrs} = map PslModel._attrName attrs
+
+    -- We define a primary field as a field that has the @id attribute.
+    attrNameAssociatedWithPrimaryField :: String
+    attrNameAssociatedWithPrimaryField = "id"
```

Comment:
> But the question about composite PK is a good question! Would be cool if we can handle that. Or, we should at least throw a nice error in that case, or something.

I've researched it a bit, and composite primary keys are defined by using block level annotations `@@id` https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#id-1 

In the current version, we'll show an error since there is no `@id` marked field. 

For example:
```wasp
entity Animal {=psl
  name String
  age Int

  @@id ([name, age])
psl=}

crud animals {
  entity: Animal,
  operations: {
    getAll: {},
  },
}
```
will produce the following:
```
- Entity referenced by "animals" CRUD declaration must have a primary key.
```

Things I could do:
1. Change the error message to always say `Entity referenced by "animals" CRUD declaration must have a primary key. (Composite primary keys are not supported yet)` 
2. Check for `@@id` annotation and then alter the message
3. Keep it as is

Reply:
Sounds good, I would say let's do (2) or (1)! And we could also add this as task under GH issue(s) for further development of CRUD.